internal/shellgen/flake_plan.go

```diff
@@ -3,13 +3,17 @@ package shellgen
 import (
 	"context"
 	"runtime/trace"
+
+	"go.jetpack.io/devbox/internal/nix"
 )
 
 // flakePlan contains the data to populate the top level flake.nix file
 // that builds the devbox environment
 type flakePlan struct {
 	NixpkgsInfo *NixpkgsInfo
 	FlakeInputs []*flakeInput
+	Packages    []*flakePackage
```

Comment:
Could this be a slice of values instead? It doesn't look like they need to be pointers.

Reply:
Can it be `nix.Packages` instead?